api/tests/test_utils.py

```diff
@@ -0,0 +1,35 @@
+from src.utils import check_mime
+
+
+def test_check_mime():
+    authorized_mime = [
+        "video/mp4",
+        "video/mpeg",
+        "video/ogg",
+        "video/webm",
+        "image/jpeg",
+        "image/png",
+        "image/bmp",
+        "image/gif",
+        "image/tiff",
+        "image/webp",
+    ]
+
+    ex_unauthorized_mime = [
+        "audio/aac",
+        "video/x-msvideo",
+        "application/pdf",
+        "application/xml" "application/vnd.mozilla.xul+xml",
+        "application/zip",
+        "video/3gpp2",
+        "audio/3gpp2",
+        "application/x-7z-compressed",
+    ]
+
+    assert any(item in authorized_mime for item in ex_unauthorized_mime) == False
+
+    for item in authorized_mime:
+        assert check_mime(item)
+
+    for item in ex_unauthorized_mime:
+        assert check_mime(item) == False
```

Comment:
You should use ` not check_mime(item)` instead of `== False`

You could also use `pytest.mark.parametrize` (https://docs.pytest.org/en/6.2.x/parametrize.html) to generate as many tests as inputs to test (here, authorized and unauthorized). It often helps to see the potential failing tests.
But it is not really necessary here.

Reply:
Thank you for your feedback! The code has been modified according to your suggestions.